apps/ensadmin/src/components/recent-registrations/components.tsx

```diff
@@ -1,41 +1,68 @@
 "use client";
 
+import { ErrorInfo, ErrorInfoProps } from "@/components/error-info";
+import { RegistrationCard } from "@/components/recent-registrations/registration-card";
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { useRawConnectionUrlParam } from "@/hooks/use-connection-url-param";
+import { cn } from "@/lib/utils";
 import type { ENSNamespaceId } from "@ensnode/datasources";
 import {
-  type ENSIndexerOverallIndexingCompletedStatus,
-  type ENSIndexerOverallIndexingFollowingStatus,
+  ENSIndexerOverallIndexingStatus,
   type ENSIndexerPublicConfig,
+  OverallIndexingStatusId,
   OverallIndexingStatusIds,
 } from "@ensnode/ensnode-sdk";
-import { fromUnixTime } from "date-fns";
+import { useAutoAnimate } from "@formkit/auto-animate/react";
+import Link from "next/link";
 import { useEffect, useState } from "react";
-
-import { Duration, RelativeTime } from "@/components/datetime-utils";
-import { NameDisplay, NameLink } from "@/components/identity/utils";
-import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
-import {
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableHeader,
-  TableRow,
-} from "@/components/ui/table";
-import { Identity } from "../identity";
 import { useRecentRegistrations } from "./hooks";
-import type { Registration } from "./types";
 
 /**
  * Max number of latest registrations to display
  */
-const MAX_NUMBER_OF_LATEST_REGISTRATIONS = 5;
+const DEFAULT_MAX_ROWS = 25;
```

Comment:
```suggestion
const DEFAULT_MAX_RECORDS = 25;
```

Please see related comments.